From 87fc7761cff5931a58984c7f7e78f421a0660e0e Mon Sep 17 00:00:00 2001
From: Tom Hochstein <tom.hochstein@nxp.com>
Date: Wed, 22 Apr 2020 13:40:04 -0500
Subject: [PATCH] meson: Add missing wayland dependency on EGL

The build was not finding egl.pc due to this
missing dependency.

Upstream-Status: Pending

Signed-off-by: Tom Hochstein <tom.hochstein@nxp.com>

---
 meson.build | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

Index: git/meson.build
===================================================================
--- git.orig/meson.build
+++ git/meson.build
@@ -65,6 +65,7 @@ dep_gl = _dep_null
 dep_egl = _dep_null
 dep_wayland_client = _dep_null
 dep_wayland_egl = _dep_null
+dep_wayland_wayland_egl = _dep_null
 dep_wayland_scanner = _dep_null
 dep_wayland_proto = _dep_null
 dep_x11_xcb = _dep_null
@@ -106,7 +107,8 @@ else
   dep_wayland_client = dependency(
     'wayland-client', version : '>= 1.10', required : get_option('wayland'),
   )
-  dep_wayland_egl = dependency(
+  dep_wayland_egl = dependency('egl', required : get_option('wayland'))
+  dep_wayland_wayland_egl = dependency(
     'wayland-egl', version : '>= 9.1', required : get_option('wayland'),
   )
   dep_wayland_scanner = dependency(
@@ -124,7 +126,7 @@ else
     wayland_xdg_shell_xml = join_paths(dep_wayland_proto.get_variable(pkgconfig: 'pkgdatadir'),
     'stable/xdg-shell/xdg-shell.xml')
   endif
-  build_wayland = dep_egl.found() and dep_wayland_client.found() and dep_wayland_egl.found() and dep_wayland_scanner.found() and dep_wayland_proto.found()
+  build_wayland = dep_egl.found() and dep_wayland_client.found() and dep_wayland_egl.found() and dep_wayland_scanner.found() and dep_wayland_proto.found() and dep_wayland_wayland_egl.found()
 
   # We're interested only in the headers provided
   # FINISHME: make x11_xcb and udev compile-only dependency
